app/services/start_price_thresholds.rb

```diff
@@ -0,0 +1,8 @@
+class StartPriceThresholds < Struct.new(:start_price)
```

Comment:
I think this could be in `app/models` - I'd expect most classes in `app/services` to have a `perform` method that updates or creates something